zerver/migrations/0574_backfill_directmessagegroup_group_size.py

```diff
@@ -0,0 +1,72 @@
+# Generated by Django 5.0.6 on 2024-08-16 06:44
+
+from django.db import migrations, transaction
+from django.db.backends.base.schema import BaseDatabaseSchemaEditor
+from django.db.migrations.state import StateApps
+from django.db.models import Count, Model, OuterRef, Subquery
+
+BATCH_SIZE = 1000
```

Comment:
I saw `BATCH_SIZE` being 5000/10000 in certain backfill migrations files. I guess 1000 should be fine?

Reply:
Or I guess 5000 can be better because a batch of 1000 recipient_ids not just includes direct message groups, but also of streams and 1:1 dms?

Reply:
1000 is fine. The latency for doing an extra query is pretty low, such that any batch size bigger than a few hundred is going to solve the problem of "Doing 100k individual queries in a row is expensive, when doing 100 batches would not be".

And then the other risk is that individual batches are too expensive; there, 1k seems safe; 5k might be too, but I don't see any benefit to pushing it given the size of this table.